internal/pkg/errors/error_message.go

```diff
@@ -66,7 +66,8 @@ const (
 	EnvNotFoundErrorMsg            = `environment "%s" not found`
 	OrgResourceNotFoundSuggestions = "List available %[1]ss with `confluent %[1]s list`."
 	EnvSwitchErrorMsg              = "failed to switch environment: failed to save config"
-	EnvNotSetErrorMsg              = "this command requires an environment; no environments found"
+	EnvNotSetErrorMsg              = "an environment is required; no environments found"
```

Comment:
I feel like erroring with "no environment found" implies that an environment was required.
```suggestion
	EnvNotSetErrorMsg              = "no environment found"
```